react/week-2/lesson.md

```diff
@@ -429,6 +437,57 @@ To help us cleanly split up code that performs business logic from code that sho
 
 Container components usually have some state and handler methods. Because of this they must use the `class` syntax. Presentational components on the other hand don't require the more verbose syntax. Instead they usually use the functional syntax.
 
+## Fetching data in React
+
+Often when you create a React app, you will want to get data from an API, and display it inside your components.
+How do we do this in React? Where does the API call go, and when should we trigger it?
+
+**Where:** Most likely at the top of the component tree, inside a 'container' component as explained above. You can then flow the data down into your child components as props.
+**When:** When the component is first loaded into the DOM. We call this 'mounting'.
+**How:** With a handy new hook called `useEffect`
+
+### The `useEffect` hook
+
+Just like `useState`, the `useEffect` hook is a special function that all function components have access to. This is the syntax to follow to fetch data on mount:
```

Comment:
Good point about importing. Maybe we can reword: 

> the `useEffect` hook is a special function that all function components can import and use as needed.

Reply:
I think we should show a quick example of importing. One of the things that trips students up is forgetting to import `React` in component files (so that JSX works)